platform/darwin/src/NSBundle+MGLAdditions.m

```diff
@@ -6,12 +6,13 @@ @implementation NSBundle (MGLAdditions)
 
 + (instancetype)mgl_frameworkBundle {
     NSBundle *bundle = [self bundleForClass:[MGLAccountManager class]];
-    if (![bundle.infoDictionary[@"CFBundlePackageType"] isEqualToString:@"FMWK"] && !bundle.mgl_resourcesDirectory) {
+
+    if (![bundle.infoDictionary[@"CFBundlePackageType"] isEqualToString:@"FMWK"] && [bundle pathForResource:@"Mapbox" ofType:@"bundle"]) {
         // For static frameworks, the bundle is the containing application
-        // bundle but the resources are still in the framework bundle.
-        bundle = [NSBundle bundleWithPath:[bundle.privateFrameworksPath
-                                           stringByAppendingPathComponent:@"Mapbox.framework"]];
+        // bundle but the resources are in Mapbox.bundle.
+        bundle = [self bundleWithPath:[bundle pathForResource:@"Mapbox" ofType:@"bundle"]];
```

Comment:
Perhaps refactor this change so that we don't look for Mapbox.bundle twice. This is asynchronous disk access, so I'd want to keep a light touch.